solr/CHANGES-SOLR-9376.txt

```diff
@@ -0,0 +1,21 @@
+TO BE DROPPED INTO CHANGES.txt (and possibly `solr-upgrade-notes.adoc`?)
+FOR WHATEVER VERSION THIS ULTIMATELY LANDS IN:
+
+Upgrade Notes
```

Comment:
put in `major-changes-in-solr-9.adoc`; not its own file.

Reply:
:+1: Yes, I was drafting it there, but it was never ultimately going to stay in its own file. This is getting close enough now that I'll move it into `major-changes-in-solr-9.adoc`